src/modules/tools/ToolsDashboard.js

```diff
@@ -1,31 +1,28 @@
-import React, { Component } from 'react';
+import React from 'react';
+import { useTranslation } from 'react-i18next'
 import {
 	Container, Col, Row, Button
 } from "reactstrap";
 
 
-export default class ToolsDashboard extends Component {
+export default function ToolsDashboard(props){
 
 	/* Example of use
 
 	{"tools":[{"name":"Kibana","url":127.0.0.1/kibana,"image":"tools/kibana.svg"}]}
 
 	*/
+		const { t } = useTranslation();
+		const App = props.app;
```

Comment:
You can remove lines [16](https://github.com/TeskaLabs/asab-webui/blob/refactor/turning-ToolsDashboard-module-into-functional-compononet/src/modules/tools/ToolsDashboard.js#L16) and [17](https://github.com/TeskaLabs/asab-webui/blob/refactor/turning-ToolsDashboard-module-into-functional-compononet/src/modules/tools/ToolsDashboard.js#L17) and in the code and you can replace `App` with `props.app` and `Config` with `props.app.Config` . Its a good manor to do it this way because you want to state in the code that `app` is a parental component which is passed to a child.